test/Microsoft.AspNet.Mvc.Core.Test/SessionStateTempDataProviderTest.cs

```diff
@@ -177,6 +177,7 @@ public void SaveAndLoad_SimpleTypesCanBeStoredAndLoaded()
             // Arrange
             var testProvider = new SessionStateTempDataProvider();
             var inputGuid = Guid.NewGuid();
+            var inputDatetimeVal = new DateTime(DateTime.Now.Ticks, DateTimeKind.Local);
```

Comment:
nit @rynowak actually, was using `DateTime.MinValue` before.  (well, the default / can't-be-disabled parameterless `DateTime` constructor).  issue was the time zone, not the chosen time.